core/src/main/java/com/googlecode/psiprobe/tools/AccessorFactory.java

```diff
@@ -19,11 +19,6 @@
 public class AccessorFactory {
 
   /**
-   * Instantiates a new accessor factory.
-   */
-  private AccessorFactory() {}
```

Comment:
PR is coming with this fix.  I did review my JavaBeanTester code to see what was otherwise causing failure.  It was not internals for testing no-arg constructor.  It was actually kicking off a bug.  So I filed an issue to self there and will ignore related test case.

Rather than introduce even more changes in this area, just keeping it simple as originally intended.